src/domain/proof-of-liabilities/create-liabilities.ts

```diff
@@ -0,0 +1,75 @@
+import { LedgerService } from "@services/ledger"
+import { toSats } from "@domain/bitcoin"
+
+const randomInt = (min: number, max: number): number => {
+  return Math.floor(Math.random() * (max - min + 1)) + min
+}
+
+export const CreateLiabilities = async (
```

Comment:
this function is not creating an object with other "domain" functions so it should be `createLiabilities`

Reply:
Understood, I will make the changes